source/frontend/src/assets/scss/App.scss

```diff
@@ -1,22 +1,27 @@
 @import './styles.scss';
 
+.Toastify__toast.Toastify__toast--dark {
+  color: #fff;
```

Comment:
nit and a question: are we attempting to map hardcoded colors like this `#fff` to our new tokens (ie `bc-typography-text-color` or something similar) or is that outside the scope of this and would be handled with a new tech-debt task?